digit-ui-esbuild/products/dashboard/src/styles/input.css

```
@@ -167,17 +167,28 @@
   }
 
   .dashboard-drag-handle-title {
-    @apply tw-truncate tw-font-semibold tw-text-foreground;
-
+    @apply tw-min-w-0 tw-font-semibold tw-text-foreground;
     font-size: 18px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)